### PR TITLE
Add membership option

### PR DESCRIPTION
**Pull request by thekvs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,6 @@ doc/_build
 
 # Generated man page
 doc/gitlab_backup.1
+
+.venv/
+.vscode/
```
Comment (josegonzalez) [inline .gitignore]: Revert these changes (and squash them out of your history). Thanks.

Comment (thekvs) [reply .gitignore]: Done

**Comment by josegonzalez** [comment]
Released in 0.3.0.
